presto-orc/src/main/java/com/facebook/presto/orc/reader/ByteSelectiveStreamReader.java

```diff
@@ -45,13 +47,15 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkState;
 import static io.airlift.slice.SizeOf.sizeOf;
+import static java.lang.Math.ceil;
 import static java.util.Objects.requireNonNull;
 
 public class ByteSelectiveStreamReader
         implements SelectiveStreamReader
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(ByteSelectiveStreamReader.class).instanceSize();
     private static final Block NULL_BLOCK = TINYINT.createBlockBuilder(null, 1).appendNull().build();
+    private static final float BATCH_EXECUTION_FILTERRATE_THRESHOLD = 0.5f;
```

Comment:
Do you expect other readers to have different value for this threshold? If not, let's move it to ReaderUtils.